stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/plugin/logic/channel/internal/ChannelLogic.kt

```diff
@@ -16,6 +16,7 @@
 
 package io.getstream.chat.android.offline.plugin.logic.channel.internal
 
+import android.util.Log
```

Comment:
I wonder why we do use android Log here.
Shouldn't we use ChatLogger or StreamLog instead?

Reply:
This was just a debug log that should have been deleted.